index.html

```diff
@@ -0,0 +1,169 @@
+        No prior experience required.
+      </p>
+      <h2 class="date-set">2023.02.01(MON) - 15(THUR)</h2>
+      <p class="location">(Addis Ababa Arat Kilo Campus IT hall)</p>
+    </main>
+    <section class="main-program-list">
+      <h2 class="main-program-title">Main Program</h2>
+      <hr>
+      <section class="programs">
+          <section class="program-one">
+            <img class="program-tag" src="images/html.png" alt="Html logo">
+            <h2 class="program-title">HTML</h2>
+            <p class="program-content">Learn how to use CSS to style and visually orginize HTML pages</p>
+          </section>
+          <section class="program-two">
+            <img class="program-tag" src="images/css.png" alt="css logo">
+            <h2 class="program-title">CSS</h2>
+            <p class="program-content">Learn how to use CSS to style and visually orginize HTML pages</p>
+          </section>
+          <section class="program-three">
+            <img class="program-tag" src="images/js.png" alt="javascript logo">
+            <h2 class="program-title">JS</h2>
+            <p class="program-content">Learn how to use CSS to style and visually orginize HTML pages</p>
+          </section>
+      </section>
+      <button class="program-join-btn">Join the Class for 2023</button>
+      </section>
+
+      <h2 class="featured-speaker-title"></h2>
+
+      <section id="all-speakers">
+        
+      </section>
+      <div class="load-more">
+        <button onclick="myFunction()" class="my-btn" id="myBtn">Read more</button>
+      </div>
+     
+      
+    </section>
+
+    <section class="desktop-partner">
+      <div class="d-partner">
+        <h4 class="d-partner-title">Partner</h4>
+        <hr>
+        <ul class="d-sponsors">
+          <li class="d-mozilla">Mozilla</li>
+          <li class="d-google">Goole</li>
+          <li class="d-naver">NAVER</li>
+          <li class="d-airbnb">airbnb</li>
+          <li class="d-airoflot">Airoflot</li>
+        </ul>
+      </div>
+      <footer class="d-footer">
+        <div class="d-copy">@creative <br> Commons Kerea</div>
+        <div class="alright-reserved">
+          <p class="d-footer-content">Creative commons korea. Some rights reserved.</p>
+        </div>
+      </footer>
+    </section>
+  
+  <!-- About page section -->
+  <section id="about-course" class="about-page">
+    <div class="about-page-card">
+        <h2 class="about-page-title">
+          INPROVE YOUR LIFE THROUGH LEARNING
+        </h2>
+        <p class="about-page-content">
+          This course is designed for teachers delivering lessons at a low level
+          or equivalent, and for beginner programmers looking to start using HTML, CSS
+          and JavaScript. To take this course, you need to be familiar with basic 
+          programming structures such as loops and if statements, but you don't 
+          need any experience of HTML, CSS or JavaScript.
```

Comment:
- [ ] Kindly make sure to add the dark border that is surrounding this text on the Behance design

Your design:
![Screenshot from 2023-02-10 23-47-23](https://user-images.githubusercontent.com/97834160/218205263-0a80715c-fb88-4318-b8ef-c2b7d18e4611.png)
Behance design:
![Screenshot from 2023-02-10 23-49-03](https://user-images.githubusercontent.com/97834160/218205355-02db22bb-e1ca-4468-b26c-65df51b1f2b5.png)